src/ringmesh/geomodel/geomodel_builder.cpp

```diff
@@ -857,6 +860,29 @@ namespace
         LineDefinition cur_line_;
     };
 
+    double compute_angle_at_corner(
+        const Line2D& line1, const Corner2D& corner, const Line2D& line2 )
+    {
+        auto corner_id = corner.gmme();
+        bool corner_is_line1_first_boundary =
+            ( line1.boundary_gmme( 0 ) == corner_id );
+        auto alongside_vertex1 = corner_is_line1_first_boundary
+                                     ? line1.vertex( 1 )
+                                     : line1.vertex( line1.nb_vertices() - 2 );
+        Geometry::Segment2D line1_segment(
+            corner.vertex( 0 ), alongside_vertex1 );
```

Comment:
Is corner.vertex() still working?

Reply:
I will double check but if I well remember, it does not work

Reply:
It does not work